std_msgs/String link_name
bool[] data_frame
